helm/prometheus-rules/templates/kaas/phoenix/alerting-rules/aws-cloud-components.rules.yml

```diff
@@ -0,0 +1,32 @@
+{{- if eq .Values.managementCluster.provider.kind "capa" }}
+# This rule applies to capa management clusters only
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  creationTimestamp: null
+  labels:
+    {{- include "labels.common" . | nindent 4 }}
+  name: aws-cloud-components.rules
+  namespace: {{ .Values.namespace  }}
+spec:
+  groups:
+  - name: aws-cloud-controller-manager
+    rules:
+    - alert: FluxHelmReleaseFailed
+      annotations:
+        description: |-
+        {{`Flux HelmRelease {{ $labels.name }} in ns {{ $labels.exported_namespace }} on {{ $labels.installation }}/{{ $labels.cluster_id }} is stuck in Failed state.`}}
+        opsrecipe: fluxcd-failing-helmrelease/
+      expr: gotk_reconcile_condition{type="Ready", status="False", kind="HelmRelease", cluster_type="management_cluster", exported_namespace!="flux-giantswarm", name=~".*(aws-ebs-csi-driver|cloud-provider-aws).*"} > 0
```

Comment:
Maybe extend these to all critical ones i.e. add cilium, coredns, network-policies suffixes as well? And do it for every CAPI-based provider, not just CAPA, so we have an easy list that we can extend for the other providers?